datahub-web-react/src/app/identity/user/UserList.tsx

```diff
@@ -74,13 +80,32 @@ export const UserList = () => {
         const newRemovedUrns = [...removedUrns, urn];
         setRemovedUrns(newRemovedUrns);
         setTimeout(function () {
-            refetch?.();
+            usersRefetch?.();
         }, 3000);
     };
 
+    const {
```

Comment:
What is roles here? Is this just a set of all roles? We can also push this down into a RoleSelect component itself :)

Reply:
I added a SelectRole component we use in UserListItem!